fe/fe-core/src/main/java/com/starrocks/server/GlobalStateMgr.java

```diff
@@ -1228,7 +1239,7 @@ private void startNonLeaderDaemonThreads() {
         esRepository.start();
         starRocksRepository.start();
         // materialized view active checker
-        mvActiveChecker.start();
+        // mvActiveChecker.start();
```

Comment:
delete ?

Reply:
I will delete these code in next PR,  not only this line code  needs to delete